### PR TITLE
1624 - adding the view to preview the docket record pdf

### PR DESCRIPTION
**Pull request by codyseibert**

### CODE REVIEW

File: shared/src/proxies/public/generatePublicDocketRecordPdfProxy.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+const { post } = require('../requests');
+
+/**
+ * generateDocketRecordPdfInteractor (proxy)
+ *
+ * @param {object} providers the providers object
+ * @param {object} providers.applicationContext the application context
+ * @param {string} providers.docketNumber the docket number of the case
```
Comment (rachaelparris) [inline shared/src/proxies/public/generatePublicDocketRecordPdfProxy.js]: Update jsdoc

File: web-client/src/presenter/sequences/gotoPrintableDocketRecordSequence.js [hunk 2]
```diff
@@ -13,6 +13,6 @@ export const gotoPrintableDocketRecordSequence = [
   setCaseAction,
   generateDocketRecordPdfUrlAction,
   setPdfPreviewUrlSequence,
-  set(state.currentPage, 'PrintableDocketRecord'),
+  set(state.currentPage, 'PublicPrintableDocketRecord'),
```
Comment (rachaelparris) [inline web-client/src/presenter/sequences/gotoPrintableDocketRecordSequence.js]: Left this change in by mistake?

**Review by rachaelparris — APPROVED** [review]

File: web-client/src/views/Public/PublicPrintableDocketRecord.jsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        <PublicCaseDetailHeader />
+        <div className="grid-container print-docket-record">
+          <Button
+            link
+            className="margin-bottom-3"
+            onClick={() => {
+              navigateBackSequence();
+            }}
+          >
+            <FontAwesomeIcon icon={['fa', 'arrow-alt-circle-left']} />
```
Comment (kkoskelin) [inline web-client/src/views/Public/PublicPrintableDocketRecord.jsx]: The `Button` component can receive the icon attribute to do this for you. Just attach
`icon={['fa', 'arrow-alt-circle-left']}`
and Bob's yer uncle.

**Review by kkoskelin — APPROVED** [review]

File: web-client/src/presenter/actions/Public/generatePublicDocketRecordPdfUrlAction.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import { state } from 'cerebral';
+/**
+ * get the pdf file and pdf blob url from the passed in htmlString
+ *
+ * @param {object} providers the providers object
+ * @param {Function} providers.get the cerebral get function
+ * @returns {object} the pdfUrl
```
Comment (kkoskelin) [inline web-client/src/presenter/actions/Public/generatePublicDocketRecordPdfUrlAction.js]: description not accurate. missing `router`

File: web-client/src/views/Public/PublicDocketRecordHeader.jsx [hunk 5]
```diff
@@ -1,37 +1,41 @@
...
-              link
-              aria-hidden="true"
-              className="show-on-mobile margin-top-1 text-left"
-              icon="print"
-              onClick={() => {}}
-            >
-              Printable Docket Record
-            </Button>
+export const PublicDocketRecordHeader = connect(
+  { caseDetail: state.caseDetail },
```
Comment (kkoskelin) [inline web-client/src/views/Public/PublicDocketRecordHeader.jsx]: Might be better to connect to `state.caseDetail.docketNumber` directly.

File: web-client/src/views/Public/PublicDocketRecordHeader.jsx [hunk 6]
```diff
@@ -1,37 +1,41 @@
...
+  ({ caseDetail }) => {
+    return (
+      <React.Fragment>
+        <div className="title">
+          <h1>Docket Record</h1>
+          <Button
+            link
+            aria-label="printable docket record"
```
Comment (kkoskelin) [inline web-client/src/views/Public/PublicDocketRecordHeader.jsx]: `aria-label` here is redundant, adds nothing to the provided text on the button itself.

**Review by kkoskelin — CHANGES_REQUESTED** [review]
Noticed a few things on second glance.

**Review by kkoskelin — APPROVED** [review]

**Review by kkoskelin — APPROVED** [review]

**Pull request merged**